packages/components/FullHeightSeparator.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { View } from "react-native";
+
+import { neutral33 } from "../utils/style/colors";
+
+export const FullHeightSeparator: React.FC = () => {
```

Comment:
Seems that we have already components: `Separator`, I've just checked and see that this component can be replaced by `<Separator horizontal color={neutral33}  />`